src/app/api/v1/share-links/[id]/endpoints/[endpointId]/route.test.ts

```diff
@@ -66,16 +66,16 @@ describe('GET /api/v1/[id]/[endpointId]', () => {
     (getAccessTicketUseCase as jest.Mock).mockResolvedValue(mockTicket);
     (getSingleSHLinkUseCase as jest.Mock).mockResolvedValue(null);
 
-    const request = mockRequest('valid-ticket');
-    const response = await GET(request, { params: mockParams });
+    const request = mockRequest('123456789');
+    const response = await GET(request, { params: {id: 'abc', endpointId: ''} });
 
     expect(response).toBeInstanceOf(NextResponse);
     expect(response.status).toBe(404);
     const responseBody = await response.json();
     expect(responseBody).toEqual({ message: NOT_FOUND });
   });
 
-  it('should return 404 if user is not found', async () => {
+  it('should return 401 if user is not found', async () => {
```

Comment:
Minor: With the update from 404 to 401, should we update the test description to indicate `... if user is not authorized`?